src/main/resources/modules/gallstones.json

```diff
@@ -1252,42 +1255,50 @@
       "type": "Observation",
       "category": "procedure",
       "unit": "",
-      "codes": [{
-        "system": "SNOMED-CT",
-        "code": 29303009,
-        "display": "Electrocardiographic procedure"
-      }],
+      "codes": [
+        {
+          "system": "SNOMED-CT",
+          "code": 29303009,
```

Comment:
Surprised to see code as a number here instead of a string.

Reply:
That's a limitation of the current module builder, unfortunately. The text field doesn't have any way of indicating what type something is supposed to be, and in the case of attributes it can be anything (boolean, string, number). So for codes that look numeric they get written out as numbers.